meerk40t/gui/wxmeerk40t.py

```diff
@@ -952,8 +967,13 @@ def handleGUIException(exc_type, exc_value, exc_traceback):
         f"Python {platform.python_version()}: {platform.machine()} - wxPython: {wxversion}\n"
     )
     error_log += "".join(traceback.format_exception(exc_type, exc_value, exc_traceback))
-    print("\n")
-    print(error_log)
+    variable_info = ""
+    try:
+        frame = exc_traceback.tb_frame
+        variable_info = "\nLocal variables:\n"
+        variable_info += _variable_summary(frame.f_locals)
+    except Exception:
+        pass
```

Comment:
The f_locals in the crash frame are probably guaranteed to be safe. No real harm though.